setup.py

```diff
@@ -47,7 +47,7 @@ def run_tests(self):
     maintainer='The Autoprotocol Development Team',
     description='Python library for generating Autoprotocol',
     license='BSD',
-    maintainer_email="autoprotocol-curators@transcriptic.com",
+    maintainer_email="support@strateos.com",
```

Comment:
do we want to set up an autoprotocol.org email for this?

Reply:
yea, we don't have that setup yet... will update once we get that going